addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -295,6 +296,39 @@ func mapAntreaConfigSpec(cluster *clusterv1beta1.Cluster, config *cniv1alpha2.An
 	if semver.Compare(version, "v1.9.0") >= 0 {
 		configSpec.Antrea.AntreaConfigDataValue.FeatureGates.TopologyAwareHints = &config.Spec.Antrea.AntreaConfigDataValue.FeatureGates.TopologyAwareHints
 	}
+	// NSX related
+	if semver.Compare(version, "1.9.0") >= 0 && config.Spec.AntreaNsx.Enable {
+		configSpec.AntreaNsx.Enable = config.Spec.AntreaNsx.Enable
+		if config.Spec.AntreaNsx.BootstrapFrom.Inline != nil {
+			configSpec.AntreaNsx.BootstrapFrom.Inline.NsxManagers = config.Spec.AntreaNsx.BootstrapFrom.Inline.NsxManagers
+			configSpec.AntreaNsx.BootstrapFrom.Inline.ClusterName = config.Spec.AntreaNsx.BootstrapFrom.Inline.ClusterName
+			// NSX cert
+			secret := &corev1.Secret{}
+			err = client.Get(context.TODO(), types.NamespacedName{
```

Comment:
I'm a bit confused.

1) When `config.Spec.AntreaNsx.BootstrapFrom.Inline != nil`, I thought the nsx cert and key value are from the `config.Spec.AntreaNsx.BootstrapFrom.Inline.nsxCert."tls.crt"` and `."tls.key"`.

2) In this case, I thought it should set `config.Spec.AntreaNsx.BootstrapFrom = Inline`, but this patch treats `config.Spec.AntreaNsx.BootstrapFrom` a dict object.